tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -1079,3 +1079,122 @@ Feature: List a sharing permissions
       | drive    |
       | Personal |
       | Shares   |
+
+
+  Scenario: user lists the permissions of their personal spaces
+    When user "Alice" lists the permissions of space "Personal" using the Graph API
```

Comment:
The response is different from using `drives.root` endpoint. Needs clarifications here.

Should listing permissions of `Personal` space's root path be possible?
this step uses `drives.permissions.` : `drives/<drive-id>/items/<root-path-id>/permissions`

CC @ScharfViktor @rhafer

Reply:
using `drives.root` endpoint, it is not possible to list Personal space permissions

https://github.com/owncloud/ocis/blob/0960d7f1364d4348a9affeb38ba3fa91e13a63ef/tests/acceptance/features/apiSharingNg/listPermissions.feature#L1040-L1045